var/spack/repos/builtin/packages/py-setuptools-scm/package.py

```diff
@@ -19,7 +19,7 @@ class PySetuptoolsScm(PythonPackage):
     version('3.1.0',  sha256='1191f2a136b5e86f7ca8ab00a97ef7aef997131f1f6d4971be69a1ef387d8b40')
     version('1.15.6', sha256='49ab4685589986a42da85706b3311a2f74f1af567d39fee6cb1e088d7a75fb5f')
 
-    variant('toml', default=False, description='Build with TOML support')
+    variant('toml', default=True, description='Build with TOML support')
```

Comment:
I think it's safe to use the `py-setuptools-scm+toml` by default because it's the preferred way to configure it.
I needed to change it to resolve a concretization issue.